xml/ha_maintenance.xml

```diff
@@ -0,0 +1,548 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+ xml:id="cha.ha.maintenance">
+ <title>Executing Maintenance Tasks</title>
+ <info>
+  <abstract>
+   <para>
+    To perform maintenance tasks on the cluster nodes, you might need to stop
+    the resources running on that node, to move them, or to shut down or reboot
+    the node.
```

Comment:
I like the description :). Would it be better to add something like"
 "..., or even to stop cluster service without stopping resources."
?

, which kind of also covers the purpose of maintenance mode.